### DISCUSSION TITLE
Nuxt Bridge is not compatible with Vue 3, right?

### DISCUSSION THREAD

**Discussion by christine927t** [nuxt/bridge#525] [Q&A] [answered]
Can someone please confirm if Nuxt Bridge is compatible with Vue 3 or not? 

Backstory if it matters: I recently upgraded our Vue/Nuxt app to Nuxt Bridge following the guide: https://v3.nuxtjs.org/bridge/overview
After the upgrade was complete I ran `npm view vue version` in my project terminal and got `3.2.39`; however, the vue folder in `node_modules` still shows the vue version as `2.7.10`. From what I understand `npm view vue version` shows the globally installed version of a package, although I'm not sure how I ever got to 3.2.39. 
When I attempt to upgrade Vue to version 3 I'm not able to start the app.

**Comment by danielroe** [comment, marked as answer]
No, Nuxt Bridge is not compatible with Vue 3. It's possible you might have multiple versions of vue installed by your package manager. The key thing is which one is resolved by Bridge/Nuxt itself.

**Reply by christine927t** [reply]
Thanks so much for your quick reply. How do I tell which version is resolved by Bridge exactly?

**Reply by danielroe** [reply]
It's likely 2.7.10 as that's what you found in `node_modules/vue`. I think if it were resolving vue 3 version, it would explicitly complain.

**Reply by christine927t** [reply]
Gotcha, thanks again!

**Reply by jm-faqih** [reply]
So basically, Bridge is only to make sure the Nuxt 2 layer is fully ready with Nuxt 3 and in the migration process, we can't use the Vue 3 component, right? Because it will be a different migration process.

**Comment by jcalfee** [comment]
Is it compatible with @vue-compat as required by https://bootstrap-vue.org/vue3 ?
